skimage/transform/_geometric.py

```diff
@@ -903,7 +876,11 @@ def __call__(self, coords):
 
         return out
 
-    def inverse(self, coords):
+    @property
+    def inverse(self):
+        raise NotImplementedError()
```

Comment:
Please include a message that specifies that `inverse_map` has replaced this functionality.